grails-app/services/org/pih/warehouse/core/DocumentService.groovy

```diff
@@ -404,9 +404,11 @@ class DocumentService {
     void createExcelRow(HSSFSheet sheet, int rowNumber, Map dataRow) {
         Row excelRow = sheet.createRow(rowNumber)
         dataRow.keySet().eachWithIndex { columnName, index ->
-            def cellValue = dataRow.get(columnName) ?: ""
-            // POI can't handle objects so we need to convert all objects to strings unless they are numeric
-            if (!(cellValue instanceof Number)) {
+            def actualValue = dataRow.get(columnName)
+            // We can't just check the truthiness of the actualValue, because the false boolean would be evaluated to an empty string
+            def cellValue = actualValue == null ? "" : actualValue
+            // POI can't handle objects so we need to convert all objects to strings unless they are numeric or boolean
+            if (!(cellValue instanceof Number) && !(cellValue instanceof Boolean)) {
```

Comment:
We can still pass booleans as strings for some imports. This method is used only for exports, they don't touch the import, so nothing would be broken in terms of the import.
The only change is that now, if e.g. a location/product source is inactive, it won't be exported as an empty string, but with the actual boolean value:
![Screenshot from 2024-10-08 16-35-17](https://github.com/user-attachments/assets/c997fd51-c3b6-4a42-b247-d25e95a4ab70)
vs
![Screenshot from 2024-10-08 16-36-10](https://github.com/user-attachments/assets/718566c9-03ae-4784-8e52-8570de8a63d9)

This change actually would helps users, that they won't eventually have to fill those empty cells with `false`, as the exported file would already contain the value, that is supposed to be passed during the import.

Reply:
@awalkowiak this would affect e.g. location export, and I mentioned it in the description, but as I said, I believe this could be somewhat expected, as this would fix the same issue in the location export as well (users wouldn't have to fill empty cells with `false`), as the exported file would be compatible to be imported.
Moreover, in the ticket I had an info bar:

> Check Locations export as well, and other exports with active/inactive status

so I believe it would be expected to be done in those as well, but I haven't asked Manon explicitly about that.

TL:DR - I will ask Manon to make sure it's fine to impact/fix this issue in other exports as well and will reach out to you guys.